CBLClient/Apps/CBLTestServer-Android/app/build.gradle

```diff
@@ -12,14 +12,14 @@ ext {
 }
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
     buildToolsVersion "29.0.3"
-    ndkVersion "21.3.6528147"
+    ndkVersion "22.0.7026061"
 
     defaultConfig {
         applicationId "com.couchbase.TestServerApp"
         minSdkVersion 22
-        targetSdkVersion 28
+        targetSdkVersion 30
```

Comment:
Hope this change will compile all early SDK versions too.

Reply:
the build machine works fine, on your local, you will have to upgrade you local android studio, but google store enforce to use 29+, so it's time to set 30 for us.